securedrop/tests/test_2fa.py

```diff
@@ -1,61 +1,70 @@
 # -*- coding: utf-8 -*-
 import os
-import random
 
 from flask import url_for
-from flask_testing import TestCase
+import flask_testing
 
 os.environ['SECUREDROP_ENV'] = 'test'
-from db import db_session, Journalist, BadTokenException
+from db import Journalist, BadTokenException
 import journalist
 import utils
 
-# Smugly seed the RNG for deterministic testing
-random.seed('¯\_(ツ)_/¯')
 
-
-class TestJournalist2FA(TestCase):
+class TestJournalist2FA(flask_testing.TestCase):
```

Comment:
@fowlslegs We discussed this in person, but I'll reiterate for everyone's benefit: generally, I prefer PR's be kept focused, and tightly scoped to resolving their associated issue. Changes like this one (and many of the ones above it in this file) are basically cosmetic, and are unnecessary for resolution of the associated issue. While I understand it can be difficult to resist the urge to do cleanups "in the neighborhood" as you work on a PR, I think it's important to resist. This kind of unnecessary extra changes in a PR make it harder to review, and also make it harder to use tools like `git blame` and `git log` to reason about the history of changes to a codebase; this often makes debugging harder for people in the future.

The original PR that I approved was tightly scoped and only made changes within `TestJournalist2FA`; that is still my "ideal" PR. You made some cosmetic changes and refactors, and at my request moved most of them to a separate PR (#1823), but some of the additional unnecessary changes remain in this PR. I've agreed to merge this, but as a note for the future, and to everyone: please keep PR's tightly focused on the issue they address as much as possible/reasonable.